src/app/blog/[title]/_components/GenerateTOC.tsx

```
@@ -0,0 +1,99 @@
+            key={index}
+            className={`${liClass} leading-tight`}
+            role="menuitem"
+            onClick={() => {
+              setSelectedIndex(index);
+              document.getElementById(id)?.scrollIntoView({
+                behavior: "smooth",
+                block: "start",
+              });
+            }}
+          >
+            <a href={`#${id}`} className={aClass}>
```

Biome

[error] The HTML element with the interactive role "menuitem" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "menuitem".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)